cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -105,7 +104,7 @@ impl Into<AMode> for StackAMode {
 
 /// AArch64-specific ABI behavior. This struct just serves as an implementation
 /// point for the trait; it is never actually instantiated.
-pub struct AArch64MachineImpl;
+pub(crate) struct AArch64MachineImpl;
 
 impl ABIMachineImpl for AArch64MachineImpl {
```

Comment:
(Another belated suggestion) `Impl` in a name (here, in `ABIMachineImpl`) makes me think that we have a concrete type, while it's actually a trait. Is there any other name that could avoid this confusion? `ABIMachineSpec` for instance?

Reply:
Good idea!